test/framework/src/main/java/org/elasticsearch/common/lucene/store/ESIndexInputTestCase.java

```diff
@@ -60,26 +62,37 @@ protected byte[] randomReadAndSlice(IndexInput indexInput, int length) throws IO
         int readPos = (int) indexInput.getFilePointer();
         byte[] output = new byte[length];
         while (readPos < length) {
-            switch (randomIntBetween(0, 5)) {
-                case 0:
-                    // Read by one byte at a time
-                    output[readPos++] = indexInput.readByte();
+            final var readStrategy = randomFrom(2, 6);
+            switch (readStrategy) {
```

Comment:
Above looks suspicious. Should it be `randomIntBetween(0, 9)`?

Reply:
Yes, fix incoming in https://github.com/elastic/elasticsearch/pull/93208/files#diff-1313cb767b5d3c0840a46464b34a645015f7dffb344967cf380dba0a5f54d833R65

Reply:
and https://github.com/elastic/elasticsearch/pull/93227 fixes this in 8.6 too